packages/liveblocks-node/src/__tests__/Session.test.ts

```diff
@@ -20,17 +20,27 @@ describe("authorization (new API)", () => {
         secret: 123,
       })
     ).toThrow(
-      'Invalid value for field "secret". Secret keys must start with "sk_". Please provide the secret key from your Liveblocks dashboard at https://liveblocks.io/dashboard/apikeys.'
+      "Invalid value for field 'secret'. Secret keys must start with 'sk_'. Please provide the secret key from your Liveblocks dashboard at https://liveblocks.io/dashboard/apikeys."
```

Comment:
(Switching to single quotes in error messages while at it, as those are typically more readable when in a JSON message. We do this everywhere else too.)